cadc-registry/src/test/java/ca/nrc/cadc/reg/client/MockServerGoodCapabilitiesTests.java

```diff
@@ -0,0 +1,277 @@
+*  License, or (at your option)         licence, soit (à votre gré)
+*  any later version.                   toute version ultérieure.
+*
+*  OpenCADC is distributed in the       OpenCADC est distribué
+*  hope that it will be useful,         dans l’espoir qu’il vous
+*  but WITHOUT ANY WARRANTY;            sera utile, mais SANS AUCUNE
+*  without even the implied             GARANTIE : sans même la garantie
+*  warranty of MERCHANTABILITY          implicite de COMMERCIALISABILITÉ
+*  or FITNESS FOR A PARTICULAR          ni d’ADÉQUATION À UN OBJECTIF
+*  PURPOSE.  See the GNU Affero         PARTICULIER. Consultez la Licence
+*  General Public License for           Générale Publique GNU Affero
+*  more details.                        pour plus de détails.
+*
+*  You should have received             Vous devriez avoir reçu une
+*  a copy of the GNU Affero             copie de la Licence Générale
+*  General Public License along         Publique GNU Affero avec
+*  with OpenCADC.  If not, see          OpenCADC ; si ce n’est
+*  <http://www.gnu.org/licenses/>.      pas le cas, consultez :
+*                                       <http://www.gnu.org/licenses/>.
+*
+*  $Revision: 5 $
+*
+************************************************************************
+*/
+
+package ca.nrc.cadc.reg.client;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import static org.mockserver.model.HttpRequest.request;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.List;
+
+import org.apache.log4j.Logger;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockserver.model.ClearType;
+import org.mockserver.verify.VerificationTimes;
+
+import ca.nrc.cadc.reg.Capabilities;
+import ca.nrc.cadc.reg.Capability;
+
+/**
+ * A set of Junit tests that use a MockServer to test a how the RegistryClient handles good responses for service capabilities. 
+ * See https://www.mock-server.com/
+ * 
+ */
+public class MockServerGoodCapabilitiesTests
+extends MockServerTestBase
+    {
+    private static final Logger log = Logger.getLogger(MockServerGoodCapabilitiesTests.class);
+
+    @Before
+    @Override
+    public void setupMockServer()
+    throws IOException {
+
+        super.setupMockServer();
+        
+    }
+    
+    @Test
+    public void testSingleServiceCapabilities()
+        throws Exception {
+
+        //
+        // A single good registry.
+        RegistryClient registryClient = buildRegistryClient(
+            "http://testhost-001:1080/good-registry-001"
+            );
+        
+        //
+        // Get the good service capabilities.
+        try {
+            Capabilities capabilities = registryClient.getCapabilities(new URI("ivo://good.authority/good-service"));
+            assertNotNull(capabilities);
+            List<Capability> list = capabilities.getCapabilities();
+            assertTrue(
```

Comment:
if you use `assertEquals(2, list,size())` the failed assertion message will be more useful.

Other such assertions here (and in other test classes) could likewise be improved.